components/checkout/Confirm.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from "react";
+import Root from "../../components/common/Root";
+import Link from "next/link";
+import Head from "next/head";
+
+import { connect } from "react-redux";
+
+
+class Confirm extends Component {
+
+  render() {
+    const { orderReceipt } = this.props;
+
+    const renderPrintButton = () => {
+      if(typeof window !== 'undefined') {
```

Comment:
```suggestion
      if (typeof window === 'undefined') {
        return null;
      }
      ...
```

You can reduce your nesting a little and remove the unnecessary `<div></div>`

Reply:
awesome thank you!